backends/saves/windows/windows-saves.cpp

```diff
@@ -32,15 +32,25 @@
 
 #include "common/scummsys.h"
 #include "common/config-manager.h"
+#include "common/file.h"
 #include "backends/saves/windows/windows-saves.h"
 #include "backends/platform/sdl/win32/win32_wrapper.h"
 
+#define DEFAULT_CONFIG_FILE "scummvm.ini"
+
 WindowsSaveFileManager::WindowsSaveFileManager() {
 	char defaultSavepath[MAXPATHLEN];
 
+	// If a scummvm.ini file is present in the current directory
+	// store the savegames here
+	if (Common::File::exists(DEFAULT_CONFIG_FILE)) {
+		strcpy(defaultSavepath, "Saved games");
+		CreateDirectory(defaultSavepath, NULL);
+		ConfMan.registerDefault("savepath", defaultSavepath);
+	}
 
 	// Use the Application Data directory of the user profile.
-	if (SHGetFolderPathFunc(NULL, CSIDL_APPDATA, NULL, SHGFP_TYPE_CURRENT, defaultSavepath) == S_OK) {
+	if (!Common::File::exists(DEFAULT_CONFIG_FILE) && (SHGetFolderPathFunc(NULL, CSIDL_APPDATA, NULL, SHGFP_TYPE_CURRENT, defaultSavepath) == S_OK)) {
```

Comment:
It may be simpler to just return if the above if statement is true. Alternatively, you could make this an else if statement.